lib/Suggestions.js

```diff
@@ -81,28 +90,54 @@ class Suggestions extends Component {
     return query.length >= minQueryLength && isFocused;
   };
 
+  renderSuggestions = () => {
+    const {
+      classNames: { suggestionLabel, activeSuggestion, suggestionContent },
+      handleClick,
+      handleHover,
+      query,
+      selectedIndex,
+      suggestions,
+      renderSuggestions,
+    } = this.props;
+
+    return suggestions.map((item, i) => {
+      const suggestionLabelElement = (
+        <span
+          className={suggestionLabel}
+          dangerouslySetInnerHTML={this.markIt(item, query)}
+        />
+      );
+
+      const content = renderSuggestions
+        ? renderSuggestions(suggestionLabelElement)
```

Comment:
instead of calling function for each element in the array, can you improve this logic to actually pass array of elements so its called only once ?